lib/vanagon/component/dsl.rb

```diff
@@ -184,7 +184,8 @@ def install_service(service_file, default_file = nil, service_name = @component.
           # Return here because there is no file to install, just a string read in
           return
         when "windows"
-          # Placeholder for Windows
+          @component.service = OpenStruct.new(:name => service_name, :service_file => service_file)
```

Comment:
It'll look the same. https://gist.github.com/McdonaldSeanp/0d1973b017161d78396b is a snippet from mcollective, and there will just be another case under windows with filepaths to .exe files